instance/ssh.py

```diff
@@ -0,0 +1,23 @@
+"""
+Utilities for dealing with SSH.
```

Comment:
@kewne, since this is just one utility function, wouldn't it be simpler to add it to `instance/utils.py`

Reply:
@lgp171188 That module contains a mix of totally unrelated functions, so I think that would just add to the confusion.

I personally think that, ironically, `utils` isn't a useful name (except when someone doesn't want to think too much about a name and uses it as a fallback 😛 ).

Reply:
@kewne, I am not convinced here that this should be in a separate file since Ocim never uses SSH directly except for this functionality. So let us agree to disagree here. But since there is no issue in having it here, I am resolving this thread in the interest of wrapping this up before the sprint ends.